Sources/Dictionary/dictionaryManager.swift

```diff
@@ -0,0 +1,19 @@
+import OS
+
+public class DictionaryManager {
+	let gameWords = words
+
+	public init() {
+	}
+
+	public func getRandomIndex() -> Int {
```

Comment:
Does this method need to be `public`?  Is it intended to be used outside of the `DictionaryManager`?